### ISSUE TITLE
how to load pre-trained model?

### ISSUE THREAD

**Issue by bitwangdan** [hx173149/C3D-tensorflow#8]

**Comment by bitwangdan** [comment]
How to use a pre-trained model on my own video dataset? can you tell me ?

**Comment by bitwangdan** [comment]
@hx173149 can you help me ?

**Comment by hx173149** [comment]
README has shown the download URL of my pretrained model and it can be used immediately with the [tf.restore()](https://www.tensorflow.org/versions/r0.12/api_docs/python/state_ops.html#Saver) API, maybe you should read it more carefully...

Top-1 accuracy of 72.6% should be achieved for the validation dataset with this code and pre-trained from the sports1M model. You can download my pretrained UCF101 model and mean file from here: https://www.dropbox.com/sh/8wcjrcadx4r31ux/AAAkz3dQ706pPO8ZavrztRCca?dl=0

Cheers~
